src/pages/proposals/forms/delegate/delegate-form.tsx

```diff
@@ -15,29 +15,33 @@ import { handleTransactionError } from '../../../../utils';
 
 interface Props {
   onClose: () => void;
-  userAccount: string;
 }
 
 const DelegateVotesForm = (props: Props) => {
-  const { onClose, userAccount } = props;
-  const { setChainData, transactions } = useChainData();
+  const { onClose } = props;
+  const { setChainData, transactions, userAccount, provider } = useChainData();
 
   const [error, setError] = useState('');
   const [delegationAddress, setDelegationAddress] = useState('');
   const api3Pool = useApi3Pool();
 
   const onDelegate = async () => {
-    if (!api3Pool) return;
+    if (!api3Pool || !provider) return;
 
-    if (!utils.isAddress(delegationAddress) || delegationAddress === constants.AddressZero) {
+    const delegationTargetLookupName = await go(provider.resolveName(delegationAddress));
```

Comment:
Can we skip this if [utils.isAddress(delegationAddress)](https://docs.ethers.io/v5/api/utils/address/#utils-isAddress)? i.e. if they entered a valid Ethereum address, there is no need to do a reverse lookup like this

Reply:
Actually, do we even need the lookup here? From the docs:

```
In the ethers API, nearly anywhere that accepts an address, an ENS name may be used instead, which can simplify code and make reading and debugging much simpler.
```

Reply:
Not sure, but I'd rather be explicit about this. I however forgot to rename the variable from `delegationTargetLookupName` to `delegationTargetResolvedName`. I think it's more readable now.

Reply:
I assumed that wasn't the case because [`isAddress`](https://docs.ethers.io/v5/api/utils/address/#utils-isAddress) doesn't recognize ENS names but this looks right. Is there any harm in being explicit about it though?